website/docs/cloud-docs/api-docs/organizations.mdx

```diff
@@ -892,6 +894,183 @@ curl \
+
+`POST /organizations/:organization_name/relationships/data-retention-policy`
+
+This endpoint creates a data retention policy for an organization, or updates the existing policy. Read more about [organization data retention policies](/terraform/cloud-docs/users-teams-organizations/organizations#data-retention-policies).
+
+The endpoint can also be used to change the type of the data retention policy on the organization. Making a request with the new type of policy will automatically create a data retention policy of the new type and replace the existing organization policy. For more information on the types of data retention policies, refer to [Data Retention Policy Types](/terraform/cloud-docs/api-docs/data-retention-policies#types-of-data-retention-policies).
+
+| Parameter            | Description                                                                |
+| -------------------- | -------------------------------------------------------------------------- |
+| `:organization_name` | The name of the organization to update the data retention policy for.      |
+
+| Status  | Response                  | Reason(s)                                                      |
+| ------- | ------------------------- | -------------------------------------------------------------- |
+| [204][] | No Content                | Successfully updated the organization data retention policy.   |
+| [404][] | [JSON API error object][] | Organization not found, or user unauthorized to perform action.|
+
+### Request Body
+
+This POST endpoint requires a JSON object with the following properties as a request payload.
+
+| Key path          | Type   | Default | Description                    |
+| ----------------- | ------ | ------- | ------------------------------ |
+| `data.type`       | string |         | A [data retention policy type](/terraform/cloud-docs/api-docs/data-retention-policies#types-of-data-retention-policies) |
+| `data.attributes` | object |         | The attributes for the [chosen policy type](/terraform/cloud-docs/api-docs/data-retention-policies#types-of-data-retention-policies) |
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "type": "data-retention-policy-delete-olders",
+    "attributes": {
+      "deleteOlderThanNDays": 33
+    }
+  }
+}
+```
+
+### Sample Request
+
+```shell
+$ curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/organizations/hashicorp/relationships/data-retention-policy
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "drp-Kh74zpKVGNWbpugQ",
+    "type": "data-retention-policy-delete-olders",
+    "attributes": {
+      "delete-older-than-n-days": 33
+    },
+    "relationships": {
+      "target": {
+        "data": {
+          "id": "hashicorp",
+          "type": "organization"
+        }
+      }
+    }
+  }
+}
+```
+
+## Remove data retention policy
```

Comment:
Non-Blocking Nitpick (Same as above!): Should we titlecase this for consistency between the other headers and this one?:
```suggestion
## Remove Data Retention Policy
```